openlibrary/templates/books/daisy.html

```diff
@@ -9,59 +10,57 @@
     </div>
 
     $if not page.ocaid:
         <ul class="link">
-            <li>There isn't a DAISY file available for <a href="$page.key">$page.title</a></li>
+            <li>$_("There isn't a DAISY file available for ")<a href="$page.key">$page.title</a></li>
         </ul>
     $elif page.is_daisy_encrypted():
         <div class="message locked">
-            <h2 class="sansserif">This DAISY file is protected.</h2>
-            <p>It can only be opened on a specialized device with a key issued by the Library of Congress.</p>
+            <h2 class="sansserif">$_("This DAISY file is protected.")</h2>
+            <p>$_("It can only be opened on a specialized device with a key issued by the Library of Congress.")</p>
         </div>
         <ul class="link">
-            <li><a href="$daisy.replace('XXX', page.ocaid)"><strong>Download the DAISY.zip</strong></a> for <a href="$page.key">$page.title</a></li>
+            <li><a href="$daisy.replace('XXX', page.ocaid)">$:_('<strong>Download the DAISY.zip</strong></a> for <a href="%(url)s">%(title)s</a>', url=page.key, title=websafe(page.title))</li>
         </ul>
     $else:
         <ul class="link">
-            <li><a href="$daisy.replace('XXX', page.ocaid)"><strong>Download the DAISY.zip</strong></a> for <a href="$page.key">$page.title</a></li>
+            <li><a href="$daisy.replace('XXX', page.ocaid)">$:_('<strong>Download the DAISY.zip</strong></a> for <a href="%(url)s">%(title)s</a>', url=page.key, title=websafe(page.title))</li>
         </ul>
 
 
-    <p class="largest" style="margin-bottom: 0;"><strong>Books for people who don't read print?</strong></p>
-    <p>The <a href="//archive.org">Internet Archive</a> is proud to be distributing over 1 million books free in a format called DAISY, designed for those of us who find it challenging to use regular printed media. </p>
-    <p>There are two types of DAISYs on Open Library: <i>open</i> and <i>protected</i>. Open DAISYs can be read by anyone in the world on many different devices. Protected DAISYs
-    $if page.is_daisy_encrypted():
-        (like this one)
-     can only be opened using a key issued by the <a href="http://www.loc.gov/nls/">Library of Congress NLS program</a>.</p>
-    <p><b><i>Enjoy!</i></b></p>
+    <p class="largest" style="margin-bottom: 0;"><strong>$_("Books for people who don't read print?")</strong></p>
+    <p>$:_('The <a href="//archive.org">Internet Archive</a> is proud to be distributing over 1 million books free in a format called DAISY, designed for those of us who find it challenging to use regular printed media. ')</p>
+    <p>$:_('There are two types of DAISYs on Open Library: <i>open</i> and <i>protected</i>. Open DAISYs can be read by anyone in the world on many different devices. Protected DAISYs %(encrypted)s can only be opened using a key issued by the <a href="http://www.loc.gov/nls/">Library of Congress NLS program</a>.', encrypted=encrypted)</p>
```

Comment:
This is weird for a translator; I think we need to wrap this in a big `if`/`else` block and duplicate the text.

```suggestion
    <p>$:_('There are two types of DAISYs on Open Library: <i>open</i> and <i>protected</i>. Open DAISYs can be read by anyone in the world on many different devices. Protected DAISYs %(encrypted)s can only be opened using a key issued by the <a href="http://www.loc.gov/nls/">Library of Congress NLS program</a>.', encrypted=encrypted)</p>
```